examples/submenus.py

```diff
@@ -0,0 +1,90 @@
+"""
```

Comment:
Minor recommendation, add a shebang line at the top and make it executable by user by default so it can be ran directly on macOS and Linux, i.e.:

```Python
#!/usr/bin/env python
```